homeassistant/components/tuya/sensor.py

```diff
@@ -1301,6 +1301,25 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
         ),
         *BATTERY_SENSORS,
     ),
+    # Tank Level Sensor
+    # Note: Undocumented
+    "ywcgq": (
+        TuyaSensorEntityDescription(
+            key=DPCode.LIQUID_STATE,
+            translation_key="liquid_state",
+        ),
```

Comment:
How much do we know the possible states? Would it be possible to make this an enum (or to add enum support in the future for tuya?)

Reply:
These are basically free-for-all. There is some guidelines, but they are minimal. It depends on the manfuacter that bought these chips.